ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/NodeCvePage.tsx

```diff
@@ -43,7 +57,8 @@ function NodeCvePage() {
         CVE: [exactCveIdSearchRegex],
     });
 
-    const { page, perPage } = useURLPagination(20);
+    const { page, perPage, setPage, setPerPage } = useURLPagination(20);
```

Comment:
nit: bare magic number

```suggestion
    const { page, perPage, setPage, setPerPage } = useURLPagination(DEFAULT_PAGE_SIZE);
```

Reply:
Since the default for pages under `Vulnerabilities` seems to be 20 though, I'll make this a Container-scoped constant and push it in this PR.

Reply:
> This was bothering me a bit too. I was tempted to set the default to 20 right in the hook, but that doesn't seem to be the clear winner:
> 
> `20` - 14 occurrences `10` - 9 occurrences `50` - 1 occurrence

I the early days of the product, when we had fewer and smaller customers, we would retrieve all records of an entity and paginate on the front-end, and that where the 50 counts originate.

As we grew and our customer environments grew, and the data relationships became increasing complex, we started paginating in the API, and for performance reasons, we reduced the number per page to 20. (This started with VM 1.0 in 2020.)

I'm guessing the pages with 10 per page are also for performance reasons, but I'm not sure about those.